sphinxcontrib/images.py

```diff
@@ -346,7 +348,8 @@ def setup(app):
     global DEFAULT_CONFIG
     app.require_sphinx('1.0')
     app.add_config_value('images_config', DEFAULT_CONFIG, 'env')
-    app.connect('builder-inited', configure_backend)
+    app.connect('config-inited', configure_backend)
+    app.connect('builder-inited', builder_init)
```

Comment:
Based on the suggestions above:

```suggestion
    app.connect('config-inited', check_config)
    app.connect('builder-inited', configure_backend)
```